### ISSUE TITLE
[bug] Support of client_anonymous setting

### ISSUE THREAD

**Issue by viartemev** [centrifugal/centrifugo#304]
**Describe the bug**
[Client anonymous setting](https://centrifugal.github.io/centrifugo/server/configuration/#client_anonymous) doesn't work.
I can't connect to the server without provided JWT token.

**Versions**
Centrifugo version is 2.2.2 (go1.12.6)
Client library used is centrifuge-js of version 2.2.2
Operating system is CentOS 7

**Steps to Reproduce**
Run Centrifugo with [client_anonymous setting](https://centrifugal.github.io/centrifugo/server/configuration/#client_anonymous).
Connect from centrifuge-js to the server without providing a token.
You will get an error.

**Expected behavior**
You can connect from centrifuge-js to centrifugo server without JWT with enabled client_anonymous setting.

**Code Snippets**
```
{
  "anonymous": true,
  "client_anonymous": true,
  "namespaces": [
      {
          "name": "public",
          "anonymous": true
      }
  ]
}


var centrifuge = new Centrifuge(‘url’);
centrifuge.subscribe("public", function(message) {
    console.log(message);
});
centrifuge.connect();

Error message in centrifugo:
{"level":"info","client":"4f1a4641-420a-47dd-a5d0-b83fc63ceea1","time":"2019-10-23T06:48:16Z","message":"client credentials not found"}
```

**Comment by namreg** [comment]
Faced the same issue. Then figured out that option `--client_insecure` should be passed during Centrifugo start.  Seems it's a lack in the docs.

**Comment by FZambia** [comment]
That was an error in docs - the option was supported by Centrifuge library but not ported to Centrifugo server. Fixed with commit 22e3b58217541eb763c867e8da3e2d30c64692c2 and will be available in next release
